webpages/db_functions.php

```diff
@@ -852,6 +856,7 @@ function retrieve_session_from_db($sessionid) {
     $session["streamlink"] = $sessionarray["streaminglink"];
     $session["signlink"] = $sessionarray["signuplink"];
     $session["hashtag"] = $sessionarray["hashtag"];
+    $session["techlevelid"] = $sessionarray["techlevelid"];
```

Comment:
While I get that you're following an existing pattern, I confess that I'd favour making the database column have a default of Unspecified. That allows you to avoid having '99' be a magic number in the code.

Otherwise, you'll want to ensure that webpages/api/brainstorm/submit_session.php has a similar change to set the tech level to 99.

Reply:
Sorry: that comment should have been on `data_functions.php`

Reply:
The values set in `data_functions.php` are used to populate the dropdowns when you load the `CreateSessions.php` page. So while doing it via a database default may cause it to be set when you save, it shows up as a blank entry in the dropdown in the create screen which can be a bit confusing.

We could go interrogate the database schema for the default column values to populate the session object, but that seems beyond the scope of this PR.

Reply:
Okay, that's a fair point.